docs/api-reference/widgets/scale-widget.md

```diff
@@ -0,0 +1,61 @@
+import {WidgetPreview} from '@site/src/doc-demos/widgets';
+import {_ScaleWidget} from '@deck.gl/widgets';
+
+# ScaleWidget (Experimental)
+
+<img src="https://img.shields.io/badge/from-v9.2-green.svg?style=flat-square" alt="from v9.2" />
+
+This widget displays a dynamic scale bar (like Google Maps) that updates as the map zoom and latitude change. It renders  a horizontal line with two tick marks at the ends, and a formatted distance label.
```

Comment:
```suggestion
This widget displays a dynamic cartographic scale bar that updates as the map view changes. It shows a horizontal line with end tick marks and a distance label, reflecting the current map scale based on zoom level and latitude.
```

Let's not use commercial references unless it's necessary. I took a pass on the flow